Python/xTriggerNamedResponder.py

```diff
@@ -70,7 +70,7 @@ def __init__(self):
         PtDebugPrint("__init__xTriggerNamedResponder v. %d" % self.version)
 
     def OnNotify(self,state,id,events):
-        if state != true:
+        if state != True:
```

Comment:
I think `if not state:` would be more Pythonic. Ideally we would do `if state is False:` but I think `state` is actually a float, so this needs to change either way.